scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/synchronization/SynchronizationWrapper.java

```diff
@@ -0,0 +1,256 @@
+    public Serializable compute(String channel, String key, String remappingFunction)
+            throws InvalidChannelException, CompilationException, IOException {
+        return internalAPI.compute(originator, taskId, channel, key, remappingFunction);
+    }
+
+    @Override
+    public Serializable merge(String channel, String key, Serializable value, String remappingFunction)
+            throws InvalidChannelException, CompilationException, IOException {
+        return internalAPI.merge(originator, taskId, channel, key, value, remappingFunction);
+    }
+
+    @Override
+    public PredicateActionResult conditionalCompute(String channel, String key, String predicate,
+            String thenRemappingFunction) throws InvalidChannelException, CompilationException, IOException {
+        return internalAPI.conditionalCompute(originator, taskId, channel, key, predicate, thenRemappingFunction);
+    }
+
+    @Override
+    public PredicateActionResult conditionalCompute(String channel, String key, String predicate,
+            String thenRemappingFunction, String elseRemappingFunction)
+            throws InvalidChannelException, CompilationException, IOException {
+        return internalAPI.conditionalCompute(originator,
+                                              taskId,
+                                              channel,
+                                              key,
+                                              predicate,
+                                              thenRemappingFunction,
+                                              elseRemappingFunction);
+    }
+
+    @Override
+    public void conditionalReplaceAll(String channel, String predicate, String thenRemappingFunction)
+            throws InvalidChannelException, CompilationException, IOException {
+        internalAPI.conditionalReplaceAll(originator, taskId, channel, predicate, thenRemappingFunction);
+    }
+
+    @Override
+    public void conditionalReplaceAll(String channel, String predicate, String thenRemappingFunction,
+            String elseRemappingFunction) throws InvalidChannelException, CompilationException, IOException {
+        internalAPI.conditionalReplaceAll(originator,
+                                          taskId,
+                                          channel,
+                                          predicate,
+                                          thenRemappingFunction,
+                                          elseRemappingFunction);
+    }
+
+    @Override
+    public boolean waitUntil(String channel, String key, String predicate)
+            throws InvalidChannelException, CompilationException {
+        return internalAPI.waitUntil(originator, taskId, channel, key, predicate);
+    }
+
+    @Override
+    public PredicateActionResult waitUntilThen(String channel, String key, String predicate,
+            String thenRemappingFunction) throws InvalidChannelException, CompilationException, IOException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)